modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/utils/api.ts

```diff
@@ -158,6 +168,16 @@ export async function getAllObjectDefinitions() {
 	);
 }
 
+export function getAllObjectDefinitionsByFolderURL(
```

Comment:
Hi @naraandrade! On this case, this function returns only the URL and not the itens, because this is the URL we need to pass to the FDS on the apiURL prop. I agree that the name is confusing, maybe is the case to not make a function at all and just pass the URL directly?

Here is how it is now:

```
if (selectedFolder.externalReferenceCode) {
			url = API.getAllObjectDefinitionsByFolderURL(
				selectedFolder.externalReferenceCode
			);
		}
```

I can do like this and remove the funciton from the API:

```
if (selectedFolder.externalReferenceCode) {
			url = `/o/object-admin/v1.0/object-folders/by-external-reference-code/${selectedFolder.externalReferenceCode}/object-definitions`;
		}
```
What do you think?

Reply:
Oh, I see. Yes, the name of the function was giving the wrong impression. I think it's better to follow your idea of not creating the function and just build the URL directly like you showed.

Reply:
It make sense! I pass the URL directly then, thanks!